src/desktop.c

```diff
@@ -317,14 +325,9 @@ get_cursor_context(struct server *server)
 		ret.type = LAB_SSD_ROOT;
 		return ret;
 	}
+#if HAVE_XWAYLAND
```

Comment:
This was guarding the unmanaged tree access which doesn't exist for non-xwayland compiles

Reply:
Separate fix?

Reply:
Done